packages/fxa-content-server/app/scripts/templates/index.mustache

```diff
@@ -1,15 +1,17 @@
 <div id="main-content" class="card">
   <header>
     <h1 id="fxa-enter-email-header">
-      <!-- L10N: For languages structured like English, the second phrase can read "to continue to %(serviceName)s" -->
       {{#t}}Enter your email{{/t}}
+      <span class="service">
+        <!-- L10N: For languages structured like English, the phrase can read "to continue to %(serviceName)s" -->
         {{#isInPocketMigration}}
-            <span class="service">{{#t}}Continue to %(serviceName)s{{/t}}</span>
+          {{#t}}Continue to{{/t}}
```

Comment:
I think we could wrap this one in an unsafeTranslate since it might help give translators more context that an image is after the text.